packages/react/checkbox/src/Checkbox.tsx

```diff
@@ -168,6 +158,33 @@ CheckboxIndicator.displayName = INDICATOR_NAME;
 
 /* ---------------------------------------------------------------------------------------------- */
 
+type BubbleCheckedProps = Omit<React.ComponentProps<'input'>, 'checked'> & {
```

Comment:
nit: should this be?
```suggestion
type BubbleInputProps = Omit<React.ComponentProps<'input'>, 'checked'> & {
```

Reply:
Oops, initially called the component `BubbleChecked` and then renamed for consistency with `Slider`. Which do you prefer actually?

Reply:
I like `BubbleInput` and then it's also the same as in `Slider`